### PR TITLE
Adds translator config to the HL7 serializer 

### PR DESCRIPTION
**Pull request by MauriceReeves-usds**
This PR updates the organizations and adds the HL7 translator config to the HL7 serializer writing.

## Checklist
- [x] Tested locally?
- [x] Ran `quickTest all`?
- [x] Downloaded a file from http://localhost:7071/api/download
- [ ] Updated the release notes? 
- [ ] Added a test?
- [ ] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

### CODE REVIEW

File: prime-router/quick-test.sh [hunk 1]
```diff
@@ -277,7 +277,7 @@ fi
 if [ $RUN_OH -ne 0 ]
 then
   echo Generate fake OH data, HL7!
-  text=$(./prime data --input-fake 50 --input-schema oh/oh-covid-19 --output-dir $outputdir --target-states OH --target-counties Ashtabula --output-format HL7_BATCH)
+  text=$(./prime data --input-fake 50 --input-schema oh/oh-covid-19 --output-dir $outputdir --target-states OH --target-counties Ashtabula --output-format HL7_BATCH --suppress-qst-for-aoe)
```
Comment (MauriceReeves-usds) [inline prime-router/quick-test.sh:280]: Added flag to test QST suppression

File: prime-router/src/main/kotlin/Mappers.kt [hunk 2]
```diff
@@ -406,11 +406,35 @@ class SplitMapper : Mapper {
...
+    override fun apply(element: Element, args: List<String>, values: List<ElementAndValue>): String? {
+        if (values.isEmpty()) return null
+        val value = values.firstOrNull()?.value ?: ""
+        val delimiter = ","
+        val splitElements = value.split(delimiter)
         val index = args[1].toInt()
         if (index > splitElements.count()) error("Index $index exceeds count of split values in $value")
 
-        return splitElements[index]
+        return splitElements[index].trim()
```
Comment (MauriceReeves-usds) [inline prime-router/src/main/kotlin/Mappers.kt:437]: Fun fact our parsing for mappers doesn't like trying to pass through commas as arguments. I tried many different ways including UTF encoding and writing them in hex. Nope.

File: prime-router/src/main/kotlin/azure/BlobAccess.kt [hunk 3]
```diff
@@ -38,8 +38,9 @@ class BlobAccess(
         val outputStream = ByteArrayOutputStream()
         when (report.bodyFormat) {
             Report.Format.INTERNAL -> csvSerializer.writeInternal(report, outputStream)
-            Report.Format.HL7 -> hl7Serializer.write(report, outputStream)
-            Report.Format.HL7_BATCH -> hl7Serializer.writeBatch(report, outputStream)
+            // HL7 needs some additional configuration we set on the translation in organization
+            Report.Format.HL7 -> hl7Serializer.write(report, outputStream, report.destination?.translation)
+            Report.Format.HL7_BATCH -> hl7Serializer.writeBatch(report, outputStream, report.destination?.translation)
```
Comment (MauriceReeves-usds) [inline prime-router/src/main/kotlin/azure/BlobAccess.kt:43]: Passing in the translation object for configuration

File: prime-router/quick-test.sh (hunk 1 above)
Comment (jimduff-usds) [reply prime-router/quick-test.sh:280]: I'll look for you in old Honolulu
San Francisco or **Ashtabula**
You're gonna have to leave me, now I know
But I'll see you in the sky above
In the tall grass and the ones I love
You're gonna make me lonesome when you go

File: prime-router/src/main/kotlin/Mappers.kt (hunk 2 above)
Comment (jimduff-usds) [reply prime-router/src/main/kotlin/Mappers.kt:437]: Not surprised - we're trying to use regex's to parse our fake mapper "code", whose complexity is teetering on the edge of needing a real recursive descent parser.

**Pull request merged**